components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/model/DCRConfiguration.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.dcr.model;
+
+/**
+ * DCR Configuration model.
+ */
+public class DCRConfiguration {
+
+    private Boolean isDCRFAPIEnforced;
+    private Boolean authenticationRequired;
+    private Boolean mandateSSA;
+    private String ssaJwks;
+
+    public Boolean getMandateSSA() {
+
+        return mandateSSA;
+    }
+
+    /**
+     * Set the value of mandateSSA.
+     * We only accept "true", "false" or null as valid values for this mandateSSA field.
+     *
+     * @param mandateSSA The value to set.
+     */
+    public void setMandateSSA(Boolean mandateSSA) {
+
+        this.mandateSSA = mandateSSA;
+    }
+
+    public Boolean isFAPIEnforced() {
+
```

Comment:
shall we stick to one way of  'fapi enforced or dcrFAPIEnforced'

Reply:
Since this is inside DCRConfiguration, do we need to add `dcr` for the name?

Reply:
no,what i meant is better to stick to one name in method name and variable name.

Reply:
Fixed with [7e78ec6](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2344/commits/7e78ec6794aab26539b858e02f82f34378b3cbd6)